README.md

```diff
@@ -11,7 +11,8 @@ There's also an
 Chrome and probably Safari).
 
 #### Note:
-- `uglify-js` only supports ECMAScript 5 (ES5). 
+- `uglify-js` only supports ECMAScript 5 (ES5).
+- Support for `const` is present but incomplete, and may not be transformed properly.
```

Comment:
That's probably much too verbose for the Notes sections. Can you keep your original line and footnote the additional two lines I suggested with a hyperlink to the bottom of the README?